### DISCUSSION TITLE
Dragging Tasks with checkbox alternatives

### DISCUSSION THREAD

**Discussion by naxey** [roovo/obsidian-card-board#170] [Ideas]
This would add a third kind of kanban board besides `timeline` and `tagboard`. Maybe call it an `emoji board`.

Each list would have an assigned character (or even emoji).
Dragging one task to another would then change the character to the character of the list it's dragged to.

These are some ideas on how this could be used. 
In this example, there would be 11 kanban lists. I gave the list entries the names that could be given to the kanban lists in the settings.

```
- [ ] to-do
- [/] incomplete
- [x] done
- [-] canceled
- [?] question
- [!] important
- [I]  idea
- [*] star
- [>] forwarded
- [i] information
- [🐛] bugs
```

Dragging an item from the list `to-do` to `incomplete` would then change the checkmark in the note like this:
`- [ ] example task` --> `- [/] example task`

These settings would be needed:
- the third kanban board option to enable this
- a text box to set the characters/emojis that are used in the checkboxes and give the lists their names (like the definitions for the tags). The naming scheme mentioned above could be used here.

**Comment by roovo** [comment]
thanks @Naxy -> just a note here for others that the follow-up discussion is in #171
